### ISSUE TITLE
NLP

### ISSUE THREAD

**Issue by olmocorell** [adrianacoca/Predictweet#4]
SPACY tiene modelos ya entrenados, son redes neuronales, si ya están entrenados y puedes aplicarlos te va a dar muchísimos mejores resultados que NLTK. Creo que puedes meterte con eso. Bueno, échale un ojo a ver cuánto de complicado es.

https://github.com/explosion/spacy-models
https://spacy.io/models/en

**Comment by adrianacoca** [comment]
Yes! Me estoy mirando videos, ahora lo estoy intentando aplicar al tema topics. He sacado labels 
ej:
'DATE': 'Absolute or relative dates or periods',
 'ORG': 'Companies, agencies, institutions, etc.',
 'PERSON': 'People, including fictional',
 'MONEY': 'Monetary values, including unit',
 'NORP': 'Nationalities or religious or political groups',
 'GPE': 'Countries, cities, states',
 'CARDINAL': 'Numerals that do not fall under another type'
y sigo viendo el video para ver que mas posibilidades tiene. 

Luego lo intentaré aplicar al tema sentiment!
